pkg/validation-webhook/snapshot.go

```diff
@@ -107,9 +110,47 @@ func (a admitter) Admit(ar v1.AdmissionReview) *v1.AdmissionResponse {
 			klog.Error(err)
 			return toV1AdmissionResponse(err)
 		}
-		return decideSnapshotClassV1(snapClass, oldSnapClass, a.lister)
+		return decideSnapshotClassV1(snapClass, oldSnapClass, a.snapshotLister)
+	case GroupSnapshotV1Alpha1GVR:
```

Comment:
do we need these? there's only `v1alpha1` right now

Reply:
> do we need these? there's only `v1alpha1` right now

I think its good to have it now, so we don't have to change it all over the place when
we have newer versions. 
+ it is consistent with naming of snapshot v1 resources currently present.